xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.h

```diff
@@ -65,8 +65,27 @@ class CRenderManager
   void Render(bool clear, DWORD flags = 0, DWORD alpha = 255, bool gui = true);
   bool IsVideoLayer();
   RESOLUTION GetResolution();
-  void UpdateResolution();
+
+  /*!
+   * \brief Update resolution, in synchronous mode
+   * \param fps The framerate
+   * \param width The width
+   * \param height The height
+   * \param stereomode The stereo mode, accepted values:
+   *        left_right, right_left, top_bottom, bottom_top
+   */
+  void UpdateResolution(float fps, int width, int height, std::string& stereomode);
+
+  /*!
+   * \brief Trigger update resolution, in asynchronous mode
+   * \param fps The framerate
+   * \param width The width
+   * \param height The height
+   * \param stereomode The stereo mode, accepted values:
+   *        left_right, right_left, top_bottom, bottom_top
+   */
```

Comment:
I know this was already there but I don't like the name TriggerX. It implies you trigger something to happen right away or, in other words, synchronization. Mind changing this to something like "SignalUpdateResolution" or something similar?